x-pack/plugins/security_solution/public/network/components/query_tabs/ips_query_tab_body.tsx

```diff
@@ -21,13 +20,16 @@ const NetworkTopNFlowTableManage = manageQuery(NetworkTopNFlowTable);
 export const IPsQueryTabBody = ({
   endDate,
   filterQuery,
+  flowTarget,
```

Comment:
alphabetize

Reply:
i agree it helps with readability, there has got to be an eslint rule so that prettier can do this automatically for us, or we could make one!

Reply:
ill look to do that in a follow up pr, i always end up with more line changes in PRs because I alphabetize, drives me nuts to have an unsorted list